config/migrations/20211229090000-new-publication-status.sparql

```diff
@@ -0,0 +1,87 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX pub: <http://mu.semte.ch/vocabularies/ext/publicatie/>
+PREFIX pubstatusid: <http://kanselarij.vo.data.gift/id/concept/publicatie-statussen/>
```

Comment:
De [Vlaamse URI-standaard](https://overheid.vlaanderen.be/sites/default/files/media/documenten/ict-egov/e-governement/open-data/VlaamseURI-StandaardVoorData_V1.0.pdf?timestamp=1517955906) suggereert enkelvoudige benoemingen. We proberen dat te volgen. Merk uit `config/resources` ook op dat we voor nieuw gedefinieerde entiteiten de Themis base-url proberen te gebruiken. Graag dat ineens meenemen.